rsmtool/configuration_parser.py

```diff
@@ -195,8 +149,7 @@ def __init__(self,
         filename : str, optional, keyword-only
             The name of the configuration file.
             The file must be stored in configdir.
-            This argument is not used in RSMTool and only added for
-            backwards compatibility for the deprecated `filepath` attribute.
+            This argument is currently not used in RSMTool.
```

Comment:
Can't we just remove it?

Reply:
I'd prefer to keep it in case `.filepath` is used in ETS module.